docs/blog/20240624-Daeraxa-3kStars.md

```diff
@@ -0,0 +1,136 @@
+
+With this work either complete or underway, we made our first tagged release of Pulsar - [1.100.0](https://github.com/pulsar-edit/pulsar/releases/tag/v1.100.0-beta) - on the same day that Atom was finally sunset - the 15th of December 2022.
+
+### Website, documentation, and socials
+
+<img src="./assets/website.png" height="250" />
+
+We needed to show the world that Pulsar was a serious project in order to make sure that people looking for Atom alternatives (and those looking for VS Code or other text editor alternatives) knew we were around and were a legitimate option. To this end, we created the [pulsar-edit.dev](https://pulsar-edit.dev) website to serve as a hub for our download links, news and blog posts, documentation, and links to social platforms for people to get help.
+
+We archived a copy of the original Atom flight manual as well as creating an edited, rebranded, and updated version of it to work with our website framework to make sure that we had documentation we could link to and maintain. We made a download page for people to find the latest releases easily, rather than relying purely on a GitHub release link. This was a substantial amount of work that was split between multiple people and took a while to complete. Could we have *technically* got by without this? Probably, but we wanted to show the whole community that we were deadly serious about our commitment to the project and wanted to make sure that it came across this way to anyone who stumbled upon us.
+
+We also wanted to make sure that we were catering to as many avenues of social communication as possible. We already had the Discord server, but we didn't have anywhere that catered to longer-form or asynchronous communication. To this end, we set up a [subreddit](https://www.reddit.com/r/pulsaredit/), [forum](https://github.com/orgs/pulsar-edit/discussions) and [Mastodon account](https://fosstodon.org/@pulsaredit) to try to cover as many avenues as possible to make sure people could use their preferred method of contact (where reasonable). We later added [Lemmy](https://lemmy.ml/c/pulsaredit) as a Reddit-style option for those who like the format but would rather not use Reddit.
+
+## The future
+
+<img src="./assets/rocket.png" height="200" />
+
+That just about covers the past, and we hope this fills in some of the blanks in the project timeline up until this point. We also hope it clarifies where the project came from and how it came to be. We do not want to forget the past, Pulsar wouldn't exist without Atom, but neither do we want to be tied to it forever; we are forging our own path for the project. We need to both recognise our past as well as be confident in our future direction.
+
+For details of what has been going on in the project since we got going in earnest, you can refer to our collection of [blog posts] (https://pulsar-edit.dev/blog/). But what can we expect for the future?
```

Comment:
```suggestion
For details of what has been going on in the project since we got going in earnest, you can refer to our collection of [blog posts] (https://pulsar-edit.dev/blog/).

But what can we expect for the future?
```